test-server/test-echo.c

```diff
@@ -189,7 +192,7 @@ int main(int argc, char **argv)
 	struct libwebsocket_context *context;
 	int opts = 0;
 	char interface_name[128] = "";
-	const char *interface = NULL;
+	const char *_interface = NULL;
```

Comment:
To avoid compile errors like those:

In file included from c:\mingw\include\rpcdce.h:10:0,
                 from c:\mingw\include\rpc.h:40,
                 from c:\mingw\include\windows.h:82,
                 from W:/libwebsockets/win32port/win32helpers/gettimeofday.h:12,
                 from w:/libwebsockets/test-server/test-echo.c:37:
w:/libwebsockets/test-server/test-echo.c: In function 'main':
w:/libwebsockets/test-server/test-echo.c:195:14: error: expected identifier or '(' before 'struct'
  const char *interface = NULL;
              ^
w:/libwebsockets/test-server/test-echo.c:296:14: error: expected '{' before '=' token
    interface = interface_name;
              ^
In file included from c:\mingw\include\rpcdce.h:10:0,
                 from c:\mingw\include\rpc.h:40,
                 from c:\mingw\include\windows.h:82,
                 from W:/libwebsockets/win32port/win32helpers/gettimeofday.h:12,
                 from w:/libwebsockets/test-server/test-echo.c:37:
w:/libwebsockets/test-server/test-echo.c:369:15: error: expected expression before 'struct'
  info.iface = interface;
               ^
make[2]: *** [CMakeFiles/test-echo.dir/test-server/test-echo.c.obj] Error 1
make[1]: *** [CMakeFiles/test-echo.dir/all] Error 2
make: *** [all] Error 2